samples/net/mqtt/src/modules/transport/Kconfig.transport

```diff
@@ -34,6 +34,19 @@ config MQTT_SAMPLE_TRANSPORT_BROKER_HOSTNAME
 	string "MQTT broker hostname"
 	default "test.mosquitto.org"
 
+config MQTT_SAMPLE_TRANSPORT_USER_NAME
```

Comment:
New sample options should be documented in https://github.com/nrfconnect/sdk-nrf/blob/main/samples/net/mqtt/doc/description.rst#general-options

Reply:
fixed in https://github.com/nrfconnect/sdk-nrf/pull/12430/commits/73f0500b016429a43e976c638eba1c8a0f0dba4c.